.github/workflows/ci.yml

```diff
@@ -35,8 +35,8 @@ jobs:
     if: github.actor != 'dependabot[bot]'
     uses: ./.github/workflows/expo-eas-update.yml
     with:
-      channel: ${{ github.event_name == 'pull_request' && format('pr-{0}', github.event.number) || 'preview'}}
```

Comment:
why did you remove this?

Reply:
I checked the previous commits that triggered this workflow and this default message was not being used.